webrtc/videoroom/lib/videoroom/pipeline.ex

```diff
@@ -0,0 +1,162 @@
+defmodule VideoRoom.Pipeline do
+  use Membrane.Pipeline
+
+  alias Membrane.WebRTC.{EndpointBin, Track}
+
+  require Membrane.Logger
+
+  def start_link() do
+    Membrane.Pipeline.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  @impl true
+  def handle_init(_opts) do
+    play(self())
+    {:ok, %{tracks: %{}, endpoints_tracks: %{}}}
```

Comment:
I am wondering about `endpoints_track_ids`. It's probably not important but while reading it's not obvious that in `endpoint_tracks` there are `ids` not `%Tracks{}`.